resources/contenters/ThePirateBaySe.py

```diff
@@ -53,9 +66,6 @@ class ThePirateBaySe(Content.Content):
     def isTracker(self):
         return True
 
-    def isSearcher(self):
```

Comment:
I didn't mean to. Sorry
On Jan 14, 2016 3:42 PM, "DiMartinoXBMC" <notifications@github.com> wrote:
>
> In resources/contenters/ThePirateBaySe.py:
>
> > @@ -53,9 +66,6 @@ class ThePirateBaySe(Content.Content):
> >      def isTracker(self):
> >          return True
> >
> > -    def isSearcher(self):
>
> why did you delete that?
>
> —
> Reply to this email directly or view it on GitHub.